tests/abstract_test.py

```diff
@@ -190,6 +190,45 @@ def define_dynamics(dom):
     sys = hybrid.LtiSysDyn(A, B, E, K, U, W, dom)
     return sys
 
+def define_dynamics_dual():
```

Comment:
Thanks for the additions. The code is not far from following [PEP 8](https://www.python.org/dev/peps/pep-0008/), though there are a few small differences, for example 2 lines needed between functions. The package [`autopep8`](https://pypi.python.org/pypi/autopep8) is useful for making (most of) these changes automatically.